web/pdf_page_view.js

```diff
@@ -582,6 +603,50 @@ var PDFPageView = (function PDFPageViewClosure() {
 
       return result;
     },
+
+    paintOnSvg: function PDFPageView_paintOnSvg(canvasWrapper) {
+      if (typeof PDFJSDev !== 'undefined' &&
```

Comment:
If we make this a viewer preference, we may not need this. In that case, we may also remove https://github.com/yurydelendik/pdf.js/blob/4f51d694c1f8792ed98d56f4ef17fb80bd8754ba/src/display/svg.js#L27.